guides/v2.1/javascript-dev-guide/widgets/jquery-widgets-about.md

```diff
@@ -43,5 +44,6 @@ Magento out of the box does not contain jQuery UI styles. Also, it is not recomm
 [Navigation widget]: {{page.baseurl}}/javascript-dev-guide/widgets/widget_navigation.html
 [Prompt widget]: {{page.baseurl}}/javascript-dev-guide/widgets/widget_prompt.html
 [QuickSearch widget]: {{page.baseurl}}/javascript-dev-guide/widgets/widget_quickSearch.html
+[RedirectUrl widget]: {{page.baseurl}}/javascript-dev-guide/widgets/widget_redirectUrl.html
```

Comment:
Hi @dobooth 
No, it is not correct.
Looks like we accidentally added the redirect and it might be the cause of the issue
https://github.com/magento/devdocs/pull/4715/commits/28b384c1924564e52a70cac8503d4b8aa66dcd0a#diff-567441ce0ce009fcad49b3b175c3ad13R5
We have removed the redirect and the issue should disappear.
@dobooth can you, please, check it?
Thank you!